libraries/tools/kotlin-symbol-processing-api/build.gradle.kts

```diff
@@ -38,8 +38,16 @@ sourceSets {
 }
 
 dependencies {
-    implementation(kotlinStdlib())
-    compileOnly(kotlinBuiltins())
```

Comment:
Same here.

Reply:
Yes. The API definition (should) only depends on kotlin standard library.